frontend/src/app/app.component.spec.ts

```diff
@@ -21,11 +21,4 @@ describe('AppComponent', () => {
     const app = fixture.componentInstance;
     expect(app.title).toEqual('heartbeat-frontend');
   });
-
-  it('should render title', () => {
```

Comment:
just wonder why this test case be removed

Reply:
I didn't see anywhere set "heartbeat-frontend app is running!" as the text of a span element which under "content" class. I assume its a legacy when initiating the project and now it's outdated.